comms/src/message/envelope.rs

```diff
@@ -130,20 +121,22 @@ impl MessageEnvelope {
     }
 
     /// Returns the Message deserialized from the body frame
-    pub fn message_body(&self) -> Result<Message, MessageError> {
+    pub fn to_message(&self) -> Result<Message, MessageError> {
```

Comment:
Nit: We have you renamed this? To_XX makes me think you are converting the body, which you are not doing.

Reply:
That's exactly what i'm doing ;) (deserializing the body)

Reply:
I've changed the method names to `deserialize_xxx` (header, message, encrypted_message)